lib/tasks/seed_vacancies_from_api.rake

```diff
@@ -0,0 +1,16 @@
+require 'teaching_vacancies_api'
+
+namespace :data do
+  desc 'Seed from the Teaching Vacancies API'
+  namespace :seed_from_api do
+    task vacancies: :environment do
+      next if Rails.env.production?
+
+      Rails.logger.debug("Seeding vacancies from Teaching Vacancies API in #{Rails.env}")
+      job_postings = TeachingVacancies::API.new.jobs(limit: 50)
```

Comment:
Do we need to specify the limit here, as it's already 50 by default? I'd probably leave the limit here, and set a default limit to be lower, just to be explicit

Reply:
Makes sense, I wondered whether to have a limit at all but I've set the default limit to be lower 👍